tensorflow/lite/schema/schema.fbs

```diff
@@ -566,6 +566,9 @@ table Conv2DOptions {
   fused_activation_function:ActivationFunctionType;
   dilation_w_factor:int = 1;
   dilation_h_factor:int = 1;
+  // Parameters for Conv2D version 7 or above.
+  // If set, use this for bias accumulator.
```

Comment:
"bias and accumulator" here and otheres.

Reply:
Thanks. Updated.